.github/workflows/behat.yml

```diff
@@ -0,0 +1,282 @@
+name: Behat
+
+on:
+  pull_request:
+    branches: [ "main", "staging"  ]
+  push:
+    branches: [ "vincetheis-behat" ]
```

Comment:
Hvad gør denne her?

Reply:
Make sure it runs on main and staging, and then it also runs on that behat banch which is our github actions debug branch.

It doesn't so much outside of debugging.